### ISSUE TITLE
Ashwalker Rework (Make Ashwalkers a Neutral Mob)

### ISSUE THREAD

**Issue by Smudged** [yogstation13/Yogstation#2877]
I want to change the ashwalkers from an instantly valid, instantly hostile mob and shift it to more of a neutral mob, like the free-golems or pod-people to add more variety to lavaland besides "Unga me ashwalker time stab" 

- Add back the Ashwalker Chief, who'll know common to be able to talk to miners and try to negotiate with them if they so please. 

- Only the Chief knows how to use more advanced technology, and ashwalkers can only speak draconic as per usual and they cannot interact with more advanced technology, so they do rely on their chieftain.

-Depending on how strong ashwalkers become remove their ash resistance (Still circumvented by pathfinder gear) Side note: I'm against removing their ash resistance, but it might be a good change.

- Gives Ashwalkers the ability to make buildings and structures from wood, and gives them the ability to also farm wood (Sprite work will be needed for these structures, most likely walls, and if there is REALLY incentive, furniture.

- Maybe make these structures the ability to make area's ash proof as well.

- Add ruling so they have the same rules as a neutral mob.

- Change Ashwalker Flavor text to reflect these changes.

- Keep Tendril the way it is, ashwalkers are now given more incentive to gather other flesh from the local fauna or trade with the cargo or the station for monkies and cows to make more ashwalkers.

**Comment by ThatLing** [comment]
I'm okay with these changes, I will bring this up with the council/staff. Will you code this yourself?

**Comment by Alexandria404** [comment]
As long as rules are set in place to actually make them act neutral, sure.

**Comment by thehighwayman** [comment]
I'm fine with it

**Comment by Ktlwjec** [comment]
Ashwalker chief I would love to see brought back, as well as making them be able to understand normal and ashwalker speak, and only having the chief know how to use consoles (so like before we rebased).
I'm unsure if it's still in, but there was an option in the crafting menu for shelters made from sinew, etc. We could be thinking of small buildings like this to craft and readd that one and others in.
Doesn't make sense to me that ashwalkers have their ash resistance removed, as you know, they're called ashwalkers.
I don't like the idea of making them a neutral mob, due to them having to gib bodies to expand the tribe. It doesn't make sense to me to remove a source of this. Perhaps a change like removing their knowledge of where the lavaland base is would help a little, and the chief being the only one who knows how to use consoles to actually get up to the station.

**Comment by Smudged** [comment]
> I'm okay with these changes, I will bring this up with the council/staff. Will you code this yourself?

Unfortunately with my scheduel being as full as it is, I'll  need someone else to code it sadly. 
Until I can learn and have the attention span to learn coding and maybe even a little spriting myself, It'll have to be coded by some one else in the mean time. 
That or, I'll just need to wait till I can find time to work on the project if I ever learn how to code properly which is fine too. 

>Doesn't make sense to me that ashwalkers have their ash resistance removed, as you know, they're called ashwalkers.

The only reason I was thinking about removing their ash resistance was because with this change they might become stronger, but you also do make a good point that they are called "Ash" Walkers.
This is where my idea of shelter where they can make their own ash proof buildings to expand their tribes like making little huts around would come into affect as an idea. They'd have more leeway into "Not" building these buildings with resistance, but it's more of a convenience thing at that point, random shelters dotted around for miners, little tents outside the mining base, etc. Again all for the sake of variety.

>I don't like the idea of making them a neutral mob, due to them having to gib bodies to expand the tribe. It doesn't make sense to me to remove a source of this. Perhaps a change like removing their knowledge of where the lavaland base is would help a little, and the chief being the only one who knows how to use consoles to actually get up to the station.

I made the idea to make them neutral so we don't have a player controlled mob that's "Instantly valid" 100% of the time with no gameplay interaction to them besides, killing them. I found that only boils down to give the ash walkers the ability to either A: Murderbone the entire mining team and the station shortly after or B: Have the miners slaughter an ashwalker because he was "dangerous" Which is often the case but these at least change ashwalkers to be more similar to Free golems, and I think that gives them more game play variety for interaction sake, like having more negative actions against miners that previously attacked them, or more positive if they tried to communicate. 
The whole Tendril aspect for me is. . . It is a problem in that regard, but I would like to think by making them neutral we'd give them a chance to use other methods to grow their numbers, like trading with the station for monkies, or make them hunt for lavaland mobs maybe even with miners, and that was my major intent. That they exist as a truly "tribal" race of ash lizards, and thus would have varying feelings depending on how they were treated by outsiders. The Tendril doesn't need to be inherently evil and it doesn't need removal, but more cooperation between ashwalkers, their chieftain and miners would be interesting in my eyes. More positive or "neutral" text or ashwalker memories will need to be changed to reflect these changes, and the rules section as well.
